echo/echo.go

```diff
@@ -0,0 +1,82 @@
+package echo
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/url"
+	"os"
+	"path/filepath"
+
+	"golang.org/x/xerrors"
+
+	"github.com/aquasecurity/vuln-list-update/utils"
+)
+
+const (
+	echoDir            = "echo"
+	advisoriesURLBase  = "https://advisory.echohq.com"
+	advisoriesFilePath = "data.json"
+)
+
+type option func(c *Updater)
+
+func WithVulnListDir(v string) option {
+	return func(c *Updater) { c.vulnListDir = v }
+}
+
+func WithBaseURL(v *url.URL) option {
+	return func(c *Updater) { c.baseURL = v }
+}
+
+type Updater struct {
+	vulnListDir string
+	baseURL     *url.URL
+	filePath    string
```

Comment:
I think after we removed `WithFilePath` we can also remove that field and use const.

Reply:
Done